### PR TITLE
Add formatting to images in documentation and replace string datatype to const for guildId

### PR DESCRIPTION
**Pull request by KamalikaSene**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,35 +18,47 @@ The Ballerina Discord connector offers APIs to connect and interact with the [Di
 Follow these steps to create a Discord developer account.
 
 ### Step 1: Login to Discord developer page
-
+<br/>
```
Comment (NipunaRanasinghe) [inline README.md]: No need to add line break elements and having an indentation (similar to https://github.com/ballerina-platform/module-ballerinax-zendesk/blob/main/ballerina/Module.md?plain=1) should be enough.

File: examples/automated-event-reminders/main.bal [hunk 2]
```diff
@@ -22,7 +22,7 @@ configurable string clientId = ?;
 configurable string clientSecret = ?;
 configurable string[] scopes = ?;
 
-string guildId = "YOUR_GUILD_ID";
+const guildId = "YOUR_GUILD_ID";
```
Comment (NipunaRanasinghe) [inline examples/automated-event-reminders/main.bal]: Convention is to use uppercase when defining constants.
```suggestion
const GUILD_ID = "YOUR_GUILD_ID";
```

File: examples/automated-role-assignment/main.bal [hunk 3]
```diff
@@ -21,8 +21,8 @@ import ballerinax/discord;
 import ballerina/http;
 import ballerina/time;
 
-const string CHANNEL_ID = "YOUR_CHANNEL_ID";
-const string GUILD_ID = "YOUR_GUILD_ID";
+const CHANNEL_ID = "YOUR_CHANNEL_ID";
+const GUILD_ID = "YOUR_GUILD_ID";
```
Comment (NipunaRanasinghe) [inline examples/automated-role-assignment/main.bal:25]: same as above

File: README.md (hunk 1 above)
Comment (KamalikaSene) [reply README.md]: All changes made

Comment (NipunaRanasinghe) [reply README.md]: @KamalikaSene it seems this is still not addressed

File: README.md [hunk 4]
```diff
@@ -19,33 +19,33 @@ Follow these steps to create a Discord developer account.
 
 ### Step 1: Login to Discord developer page
 
-<img src="https://github.com/ballerina-platform/module-ballerinax-discord/blob/main/docs/setup/resources/discord-dev-page.png?raw=true" alt="Discord Dev Page" style="width: 70%;">
+   <img src="https://github.com/ballerina-platform/module-ballerinax-discord/blob/main/docs/setup/resources/discord-dev-page.png?raw=true" alt="Discord Dev Page" style="width: 70%;">
```
Comment (NipunaRanasinghe) [inline README.md]: Lets have 4-spaced indentations for image links. Update all the similar places.
```suggestion
    <img src="https://github.com/ballerina-platform/module-ballerinax-discord/blob/main/docs/setup/resources/discord-dev-page.png?raw=true" alt="Discord Dev Page" style="width: 70%;">
```

File: README.md (hunk 1 above)
Comment (KamalikaSene) [reply README.md]: Changes have now been made

Comment (KamalikaSene) [reply README.md]: @NipunaRanasinghe Changes have now been made.

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**